src/Shared/FrameworkLocationHelper.cs

```diff
@@ -1126,6 +1123,25 @@ public string WindowsSdkRegistryInstallationFolderName
             {
                 get { return _windowsSdkRegistryInstallationFolderName; }
             }
+
+            /// <summary>
+            /// The key in registry to indicate the corresponding .net framework in this visual studio.
+            /// i.e. 'v8.0A' for VS11.
+            /// </summary>
+            public string GetDotNetFrameworkSdkRegistryKey(Version dotNetSdkVersion)
+            {
+                string sdkVersionFolder = "4.6"; // Default for back-compat
+
+                // Framework 4.6.1 
+                if (dotNetSdkVersion == dotNetFrameworkVersion461)
+                {
+                    sdkVersionFolder = "4.6.1";
+                }
+
+                // If the path is formatted to include a version number, we need to include that.
+                // (e.g. NETFXSDK\{0} should be NETFXSDK\4.6 or NETFXSDK\4.6.1
+                return string.Format(_dotNetFrameworkSdkRegistryKey, sdkVersionFolder);
```

Comment:
Comment that most strings don't have the format specifier and come out unmodified.